CONTRIBUTING.md

```diff
@@ -173,3 +173,40 @@ Before merging a PR make sure:
 
 Merge the PR by using "Squash and merge" option on Github. Avoid creating merge commits.
 After the merge make sure referenced issues were closed.
+
+### Developing guidelines
+
+#### Building [OCI Images](https://github.com/opencontainers/image-spec/blob/master/spec.md) for multiple arch (linux/arm64, linux/amd64)
+
+OCI images could be built and published by [buildx](https://github.com/docker/buildx), it could be executed for local test via:
+
+```
+$ OPERATOR_VERSION=devel ./.ci/publish-images.sh
```

Comment:
This isn't the operator repository ;-) The instructions here need to fit this repo.